### PR TITLE
fix(docs): add redirects to SDK docs IA change

### PR DESCRIPTION
**Pull request by dannysheridan**
This reverts commit 8713b7b7e8325dc85c56501d57894da3631272e0.

### CODE REVIEW

File: fern/docs.yml [hunk 1]
```diff
@@ -204,8 +232,8 @@ navigation:
         contents:
           - page: Overview
             path: ./pages/sdks/introduction/overview.mdx
-          - page: Configuration
-            path: ./pages/sdks/introduction/configuration.mdx
+          - page: Language Support
+            path: ./pages/sdks/introduction/language-support.mdx
```
Comment (dsinghvi) [inline fern/docs.yml:236]: we should add a redirect for this

File: fern/docs.yml [hunk 2]
```diff
@@ -227,97 +255,101 @@ navigation:
...
-          - page: RubyGems (Ruby)
-            slug: rubygems
-            path: ./pages/sdks/integration/rubygems.mdx
-          - page: Packagist (PHP)
-            slug: packagist
-            path: ./pages/sdks/integration/packagist.mdx
-
+          - page: Configuration
+            path: ./pages/sdks/introduction/configuration.mdx
+      - section: Guides
```
Comment (dsinghvi) [inline fern/docs.yml:322]: Guides gets pretty hidden, thoughts on moving this above capabilities

File: fern/docs.yml [hunk 3]
```diff
@@ -227,97 +255,101 @@ navigation:
...
+        contents:
+          - page: Generate your First SDK
+            path: ./pages/sdks/getting-started/generate-your-first-sdk.mdx
+          - page: Preview your SDK Locally
+            path: ./pages/sdks/getting-started/preview-your-sdk-locally.mdx
+          - page: Publish a Public-Facing SDK
+            path: ./pages/sdks/publish-sdk/publish-your-sdk.mdx
+            hidden: true
```
Comment (dsinghvi) [inline fern/docs.yml:330]: these links also 404ed -- probably want redirects for these

File: fern/docs.yml [hunk 4]
```diff
@@ -227,97 +255,101 @@ navigation:
...
+                path: ./pages/sdks/integration/pkgsite.mdx
+              - page: Maven Central (Java)
+                slug: maven-central
+                path: ./pages/sdks/integration/maven-central.mdx
+              - page: RubyGems (Ruby)
+                slug: rubygems
+                path: ./pages/sdks/integration/rubygems.mdx
+              - page: Packagist (PHP)
+                slug: packagist
+                path: ./pages/sdks/integration/packagist.mdx
```
Comment (dsinghvi) [inline fern/docs.yml:352]: probably want redirects for these as well

**Comment by dannysheridan** [comment]
closing this PR but will continue with this change in another, fresh PR

**Pull request closed without merging**